common/src/main/java/whocraft/tardis_refined/common/tardis/control/ConsoleControl.java

```diff
@@ -19,7 +19,8 @@ public enum ConsoleControl implements StringRepresentable {
     THROTTLE("throttle", new ThrottleControl(), "control.tardis_refined.throttle"),
     MONITOR("monitor", new MonitorControl(), "control.tardis_refined.monitor"),
     DIMENSION("dimension", new DimensionalControl(), "control.tardis_refined.dimension"),
-    FAST_RETURN("fast_return", new FastReturnControl(), "control.tardis_refined.fast_return");
+    FAST_RETURN("fast_return", new FastReturnControl(), "control.tardis_refined.fast_return"),
+    GENERIC_NO_SHOW("generic_no_show", new GenericControl(), "control.tardis_refined.generic_control");
```

Comment:
Since we are allowing for more control types, we can consider making these into a custom registry like we do for ConsoleTheme, so that other add-on mods can extend this. Using enums prevents other mods from working with our system unless they use an ugly mixing back.

Reply:
Fully behind this mindset, the way I see the mod is making it as extensible as possible

Reply:
Good thoughts, wouldn't say its for this branch.